### DISCUSSION TITLE
Scroll progress saving method

### DISCUSSION THREAD

**Discussion by SabaManjavidze** [lnreader/lnreader#208] [Q&A] [answered]
I'm curious on the reader screen about how you guys save scroll progress, what steps you take. for example, do you store it in SQLite database or do you store it in async storage? I'm not familiar with redux btw so that's the reason I'm asking

**Comment by rajarsheechatterjee** [comment, marked as answer]
Currently the app saves the scroll progress in redux state, and redux-persist saves it in async storage. I do plan to make the app directly save the progress in async storage for better performance.
